spec/models/financial_transaction_spec.rb

```diff
@@ -57,4 +57,23 @@
       expect(incidental_trans.amount).to eq(incidental.incidental_type.base+(incidental.times_modified*incidental.incidental_type.modifier_amount))
     end
   end
+
+  describe '#self.get_rental_summation' do
+    before(:each) do
+      @item_type = create :item_type, name: 'TEST_ITEM_TYPE'
+      @rental = create :rental, item_type: @item_type
+      @incidental = create :incurred_incidental, rental: @rental
+    end
+
+    it 'return the sum of financial transactions which have the same rental' do
+      result_from_function = FinancialTransaction.get_rental_summation(@rental.id)
+      sum_ft_by_rental = 0
+
+      FinancialTransaction.where(rental: @rental).each do |ft|
+        sum_ft_by_rental += ft.amount
+      end
```

Comment:
Consider using [`inject`](http://apidock.com/ruby/Enumerable/inject) to turn it into something like:
`@rental.financial_transactions.inject(0) do |memo, ft|...`


Sorry for sending you to apidock, ruby doc is down:
<img width="1157" alt="screen shot 2016-07-19 at 11 33 20 am" src="https://cloud.githubusercontent.com/assets/129601/16956180/6447e0fc-4da5-11e6-9d0b-96e461f74c71.png">